server/src/main/java/org/server/SupServer.java

```diff
@@ -186,14 +194,14 @@ private String createFormattedChatMessage(String message, String fromname, Strin
 
     /**
      * return the status to the specific one
-	 *
-	 * @param status - status info
-	 * @param toname - the specific one to transfer status
```

Comment:
my editor automatically shoots for our coding standard by doing things like replacing tabs with spaces. sorry for the extra noise.